### DISCUSSION TITLE
What is GitHub?!?!?!

### DISCUSSION THREAD

**Discussion by brialea123** [community/community#23833] [New to GitHub]
<p>Wow this is so confusing. I applied to be a content writer on LinkedIn but I’m not sure if it’s get it, I have no experience in computer software. It’s really confusing to understand and I don’t know what they talking about and it sounds like gibberish. Ugh!!!</p>

**Comment by Zerotask** [comment, marked as answer]
<p>I googled that for you.</p><aside class="onebox allowlistedgeneric" data-onebox-src="https://www.howtogeek.com/180167/htg-explains-what-is-github-and-what-do-geeks-use-it-for/">
  <header class="source">
      <img src="https://www.howtogeek.com/public/favicon.ico" class="site-icon" width="16" height="16">

      <a href="https://www.howtogeek.com/180167/htg-explains-what-is-github-and-what-do-geeks-use-it-for/" target="_blank" rel="noopener nofollow ugc">How-To Geek</a>
  </header>

  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:689/262;"><img src="https://user-images.githubusercontent.com/20150243/181079637-c2f8313a-fc23-46e6-98e9-ef5f9c9013cf.png" class="thumbnail" width="689" height="262" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/e/7/e7f670601e7249ac323ba77077bf723686acc2cd_2_689x262.png, https://aws1.discourse-cdn.com/github/optimized/3X/e/7/e7f670601e7249ac323ba77077bf723686acc2cd_2_1033x393.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/e/7/e7f670601e7249ac323ba77077bf723686acc2cd.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/e/7/e7f670601e7249ac323ba77077bf723686acc2cd_2_10x10.png"></div>

<h3><a href="https://www.howtogeek.com/180167/htg-explains-what-is-github-and-what-do-geeks-use-it-for/" target="_blank" rel="noopener nofollow ugc">What Is GitHub, and What Is It Used For?</a></h3>

  <p>GitHub is a website and service that we hear geeks rave about all the time, yet a lot of people don’t really understand what it does. Want to know what all the GitHub hubbub is about? Read on to find out.</p>

  <p>
    <span class="label1">Est. Reading Time: 3 minutes</span>
  </p>

  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<p>Alternatively you can read the wikipedia article.<br>
But if you have no clue about CS it’s probably too abstract for you. But you can learn it :wink: it’s no rocket science.</p>
